packages/grafana-runtime/src/services/ScopesSelectorContext.ts

```diff
@@ -0,0 +1,24 @@
+import { createContext, useContext, useMemo } from 'react';
+import { useObservable } from 'react-use';
+
+import { ScopesSelectorService, ScopesSelectorServiceState } from './ScopesSelectorService';
+import { getScopesSelectorService } from './scopes';
+
+export const ScopesSelectorContext = createContext<ScopesSelectorService>(getScopesSelectorService());
```

Comment:
would have expected this to be just the context & context type and the implementation to live in core (and be undefined here)